.github/workflows/ci.yml

```diff
@@ -4,6 +4,14 @@ on:
     branches:
       - main
   pull_request:
+
+concurrency:
+  # Taken from gradle/gradle
+  # On main, we don't want any jobs cancelled so the sha is used to name the group
+  # On PR branches, we cancel the job if new commits are pushed
+  group: ${{ (github.ref == 'refs/heads/main' ||  startsWith(github.ref, 'refs/tags') ) && format('contributor-pr-base-{0}', github.sha) || format('contributor-pr-{0}', github.ref) }}
```

Comment:
Just curious - have found an explanation for this kind of `if/else`?
I've tried but without success

Reply:
It's just the typescript thing with. You can join `Boolean` and other value so `true && something` will be something and similar with `||`